engine/reference/api/docker_io_accounts_api.md

```diff
@@ -4,15 +4,9 @@ aliases:
 description: API Documentation for docker.io accounts.
 keywords:
 - API, Docker, accounts, REST,  documentation
-menu:
-  main:
-    parent: engine_remoteapi
-    weight: 90
 title: docker.io accounts API
```

Comment:
Should this be '`docker.io` Accounts API'? Not sure what the proper noun is, and it's awkward to start with a lowercase word.